docs/sphinx/using/cudaq/variational.rst

```diff
@@ -94,7 +96,7 @@ Here is an example
         return cost;
       });
 
-CUDA Quantum packages the above up for the variational quantum eigensolver algorithm in 
+CUDA Quantum wraps the above code up for the variational quantum eigensolver algorithm in 
```

Comment:
Maybe "re-exports" would be a more precise verb here?

Reply:
I'm not a native English speaker, I just tripped over the formulation when reading it for the first time. I was reading "CUDA Quatnum packages" as the noun. So I'm open to any suggestions.

Reply:
```suggestion
CUDA Quantum provides the above code for the variational quantum eigensolver algorithm in 
```
I think this minor tweak will do the trick.